modules/command/src/main/java/org/apache/fluo/command/FluoExec.java

```diff
@@ -44,25 +50,39 @@ protected void configure() {
     }
   }
 
-  public static void main(String[] args) throws Exception {
-    if (args.length < 2) {
-      System.err.println("Usage: fluo exec <app> <class> args...");
-      System.exit(-1);
+  @Parameter(description = "<app> <class> args...", variableArity = true)
+  private List<String> args;
```

Comment:
It would be nice to be able to have three fields `String app`, `String class`, and `List<String> args`. However I couldn't get this to work with JCommander which is why there's just a single field `List<String> args`. I'm going to keep trying to see if I can get it to work with three fields.

Reply:
I saw a subparameter annotation that has an order parameter in the jcommander docs.   Maybe that could be used to accomplish this, not sure.

Reply:
I tried using `SubParameter` first actually, but the combination of having a main parameter with sub-parameters on a command seemed to confuse JCommander. I'll try playing around with it some more.

Reply:
Been experimenting a little more, and I briefly looked through the source code. I don't think it's possible.